ui/cypress/integration/topsql/topsql.spec.ts

```diff
@@ -167,13 +167,11 @@ skipOn(Cypress.env('TIDB_VERSION') !== 'nightly', () => {
         cy.wait('@getTopsqlSummary')
```

Comment:
Maybe you should test with seconds with sub-seconds for better coverage, e.g. now * 1000 + 456.

BTW there is indeed time range problems in Top SQL related with subseconds, as the displayed time range will vary within 59s, 60s, 61s when using the brush tool.